feat: Refactor Enable And Disable Module Commands (#1918)
kyma-project/cli: docs/gen-docs/kyma_alpha_add_module.md

---
title: kyma alpha add module
---

> Add a module in the cluster or in the given Kyma resource.

Reviewer: ```suggestion
Adds a module to the cluster or the given Kyma resource.
```

## See also

> * [kyma alpha add](kyma_alpha_add.md)	 - Enables a resource in the Kyma cluster.

Reviewer: ```suggestion
* [kyma alpha add](kyma_alpha_add.md)	 - Adds a resource to the Kyma cluster.
```
